assets/src/scss/layout/_navbar.scss

```diff
@@ -69,12 +69,27 @@ $navbar-default-height: 60px;
       }
     }
 
-    height: 44px;
-    line-height: 40px;
+    height: $sp-5x;
```

Comment:
I don't think we should use these variables for height or line-height, they are meant to be used for spacing values (so `margin`, `padding`, etc)

Reply:
Good point!